devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy

```diff
@@ -221,38 +224,33 @@ private Map parseReportAttributeMessage(String description) {
 private List parseIasMessage(String description) {
 	List parsedMsg = description.split(" ")
 	String msgCode = parsedMsg[2]
+	ZoneStatus zs = new ZoneStatus(Integer.decode(msgCode))
```

Comment:
You can replace the first three lines with `zigbee.parseZoneStatus(description)` I think